pkg/volume/plugins.go

```diff
@@ -151,6 +157,7 @@ type AttachableVolumePlugin interface {
 	VolumePlugin
 	NewAttacher() (Attacher, error)
 	NewDetacher() (Detacher, error)
+	GetDeviceMountRefs(deviceMountPath string) ([]string, error)
```

Comment:
This is reaching into volume plugins to fulfill a dependency that this class now has. But that's fine fine for now. We can come back and clean it up later (https://github.com/kubernetes/kubernetes/issues/14217).